sdk/core/core-http/src/operationOptions.ts

```diff
@@ -67,8 +67,9 @@ export function operationOptionsToRequestOptionsBase<T extends OperationOptions>
   }
 
   if (tracingOptions) {
-    result.spanOptions = tracingOptions.spanOptions;
     result.tracingContext = tracingOptions.tracingContext;
+    // Backwards compatibility
```

Comment:
Should we add more information related to which versions this might be relevant to?

Reply:
Yes absolutely, I'll add it. Great catch @sadasant